### DISCUSSION TITLE
Thanks.

### DISCUSSION THREAD

**Discussion by beqjanus** [AnimMouse/setup-rclone#4] [General]
Hi AnimMouse,
Thanks for the speedy update on the MacOS bug and for this super useful action.

I cloned this because I saw failures on MacOS in retrieving the version number. Initially, I set it up so that MacOS had a fallback default version if the version script returned null, but I was seeing Linux failures to retrieve the version last night, so I've switched to using the rclone.org "latest" URLs. I noticed this morning that you actually used to do it that way but switched away from it. Why was that? At some point, I need to revisit why my actions were failing, and ideally I can then switch back to using you main instead, but the priority was to get my builds back up and running :-)

Take care
Beq

**Comment by AnimMouse** [comment]
> Thanks for the speedy update on the MacOS bug and for this super useful action.

You're welcome, I'm so glad that my action is helping you. Can't believe that someone is running my action on macOS that I don't even bother to test it on macOS.

> I cloned this because I saw failures on MacOS in retrieving the version number.

Yes, that is because of #3, but now it is fixed on [v1.3.0](https://github.com/AnimMouse/setup-rclone/releases/tag/v1.3.0)

> I was seeing Linux failures to retrieve the version last night

Can't believe that even Ubuntu runners are getting rate limited.

> I've switched to using the rclone.org "latest" URLs. I noticed this morning that you actually used to do it that way but switched away from it. Why was that?

It is because downloading from GitHub's servers on GitHub Actions is faster than downloading on rclone.org's servers.
